src/index.js

```diff
@@ -17,33 +18,12 @@ export default function instantMeiliSearch(hostUrl, apiKey, options = {}) {
         facetsDistribution: params.facets.length ? params.facets : undefined,
         facetFilters: params.facetFilters,
         attributesToHighlight: this.attributesToHighlight,
+        attributesToCrop: params.attributesToSnippet,
         limit,
       }
       return removeUndefinedFromObject(searchInput)
     },
 
-    replaceHighlightTags: function (
-      formattedHit,
-      highlightPreTag,
-      highlightPostTag
-    ) {
-      // formattedHit is the `_formatted` object returned by MeiliSearch.
-      // It contains all the highlighted attributes
-      return Object.keys(formattedHit).reduce((result, key) => {
-        let newHighlightString = formattedHit[key] || ''
-        // If the value of the attribute is a string,
-        // the highlight is applied by MeiliSearch (<em> tags)
-        // and we replace the <em> by the expected tag for InstantSearch
-        if (isString(formattedHit[key])) {
-          newHighlightString = formattedHit[key]
-            .replace(/<em>/g, highlightPreTag)
-            .replace(/<\/em>/g, highlightPostTag)
-        }
-        result[key] = { value: newHighlightString.toString() }
-        return result
-      }, {})
-    },
-
     parseHits: function (meiliSearchHits, params) {
       if (this.pagination) {
```

Comment:
```suggestion
      if (this.pagination) {
      const {
             highlightPreTag,
            highlightPostTag,
            snippetEllipsisText,
            attributesToSnippet
            } = params;
```

Consider deconstructing objects. This helps understand sooner why `params` is used